change_color.sh

```diff
@@ -300,9 +300,10 @@ fi
 echo "== Rendering GTK 3 assets..."
 FORCE_INKSCAPE="$OPTION_FORCE_INKSCAPE" ./render-assets.sh gtk
 
-FORCE_INKSCAPE="$OPTION_FORCE_INKSCAPE" ./install.sh --dest "$TARGET_DIR" --name "${OUTPUT_THEME_NAME/\//-}" --color "$COLOR_VARIANT" --size "$SIZE_VARIANT"
+meson _build -Dprefix="$HOME/.local" -Dtheme_name="${OUTPUT_THEME_NAME/\//-}" -Dcolors="$COLOR_VARIANT" -Dsizes="$SIZE_VARIANT"
+meson install -C _build
 
-GENERATED_PATH="$DEST_PATH$(tr -d ',' <<< "$COLOR_VARIANTS")$(tr -d ',' <<< "$SIZE_VARIANTS")"
+GENERATED_PATH="$HOME/.local/share/themes/${OUTPUT_THEME_NAME/\//-}$(tr -d ',' <<< "$COLOR_VARIANTS")$(tr -d ',' <<< "$SIZE_VARIANTS")"
```

Comment:
DEST_PATH should be used here

Reply:
if you wanna change default theme destination, it's on different line:

```
TARGET_DIR=${TARGET_DIR-$HOME/.themes}
```

Reply:
Thanks. I'll change the default `TARGET_DIR` to `$HOME/.local`.

Reply:
to .local/share/themes/ in that case

as it means the parent dir for ouput theme

Reply:
Currently our meson [hard-codes the `share/themes` path](https://github.com/nana-4/materia-theme/blob/master/meson.build#L10), so the  `-Dprefix` option can't include that path. It seems meson can't get the absolute path of `$HOME` inside `meson.build` files, so we also can't do something like this:

```meson
if get_option('prefix') == '$HOME/.themes' or
   get_option('prefix') == '$HOME/.local/share/themes'
  theme_base_dir = join_paths(get_option('prefix'), theme_base_name)
endif
```

Reply:
Never mind. I'll redo this as you suggested below :)